README.md

```diff
@@ -50,6 +50,7 @@
 
```

Comment:
- [ ] It would be nice if I would be able to see the project files in the files changes in the pull request so I can proceed with review and can point out if there is something problematic in your code 👍